content/tutorials/webhook-chuck-norris.textile

```diff
@@ -1,5 +1,6 @@
 ---
 title: WebHooks and Chuck Norris
+site_title: "WebHooks and Chuck Norris"
```

Comment:
I see this is duplicated, which feels unnecessary.

Why don't we just encourage people to define a title once that works for all cases?  This approach is error prone as every new page created needs a site title defined, yet the author of this page has no way of knowing how that works with the website.